pkg/config/config_template.yaml

```diff
@@ -1194,6 +1194,11 @@ api_key:
 #
 # dogstatsd_tags:
 #   - <TAG_KEY>:<TAG_VALUE>
+#
+## @param dogstatsd_capture_path - string - optional - default: /opt/datadog-agent/run/dsd_capture (c:\ProgramData\Datadog\run\dsd_capture on Windows)
+## `dogstatsd_capture_path` defines the location of the folder where dogstatsd captures will be stored.
+#
+# dogstatsd_capture_path: /opt/datadog-agent/run/dsd_capture
```

Comment:
I don't think this is worth documenting in the config template, since I don't expect anyone to have to customize this path outside of the `run` directory. WDYT of removing this?

Reply:
In addition, for usability, I think it'd be good to allow customizing this path with a CLI flag on the command that starts a capture. I don't know if there's a technical reason that makes this difficult to implement. In any case this is not urgent and can be addressed later.

Reply:
I thought about the second point, that was a bigger change that requires modifying the API call, so I figured it made sense to change in a follow-up PR for 7.29.0.